2-mandatory/1-writers.js

```diff
@@ -16,41 +16,41 @@
 
 // We've created an array of objects for you here:
 let writers = [
-  {
-    firstName: "Virginia",
-    lastName: "Woolf",
-    occupation: "writer",
-    age: 59,
-    alive: false,
-  },
-  {
-    firstName: "Zadie",
-    lastName: "Smith",
-    occupation: "writer",
-    age: 40,
-    alive: true,
-  },
-  {
-    firstName: "Jane",
-    lastName: "Austen",
-    occupation: "writer",
-    age: 41,
-    alive: false,
-  },
-  {
-    firstName: "Bell",
-    lastName: "Hooks",
-    occupation: "writer",
-    age: 63,
-    alive: true,
-  },
-  {
-    firstName: "Yukiko",
-    lastName: "Motoya",
-    occupation: "writer",
-    age: 49,
-    alive: true,
-  }
+    {
+        firstName: "Virginia",
+        lastName: "Woolf",
+        occupation: "writer",
+        age: 59,
+        alive: false,
+    },
+    {
+        firstName: "Zadie",
+        lastName: "Smith",
+        occupation: "writer",
+        age: 40,
+        alive: true,
+    },
+    {
+        firstName: "Jane",
+        lastName: "Austen",
+        occupation: "writer",
+        age: 41,
+        alive: false,
+    },
+    {
+        firstName: "Bell",
+        lastName: "Hooks",
+        occupation: "writer",
+        age: 63,
+        alive: true,
+    },
+    {
+        firstName: "Yukiko",
+        lastName: "Motoya",
+        occupation: "writer",
+        age: 49,
+        alive: true,
+    }
```

Comment:
You do not need to modify the object. The single spacing is enough, at least, the compiler won't complain.